src/new-components/box/box.test.tsx

```diff
@@ -252,4 +253,13 @@ describe('Box', () => {
             )
         })
     })
+
+    describe('a11y', () => {
+        it('renders with no a11y violations', async () => {
+            const { container } = render(<Box>Hello world!</Box>)
+            const results = await axe(container)
+
+            expect(results).toHaveNoViolations()
+        })
+    })
```

Comment:
Is this a formality? For completeness? What's the value of testing that raw `div` renders without a11y violations?

Reply:
> Is this a formality? For completeness?

Correct.